back/src/domain/convention/useCases/agencies/UpdateAgencyStatus.unit.test.ts

```diff
@@ -83,15 +95,28 @@ describe("Update agency status", () => {
       .withStatus("active")
       .build();
 
+    it("returns 401 if no jwt payload", async () => {
+      await expectPromiseToFailWithError(
+        updateAgencyStatus.execute(
+          { id: existingAgency.id, status: "active" },
+          undefined,
+        ),
+        new UnauthorizedError(),
+      );
+    });
+
     it("returns 404 if agency not found", async () => {
       const agencyId = "not-found-id";
       await expectPromiseToFailWithError(
-        updateAgencyStatus.execute({ id: agencyId, status: "active" }),
+        updateAgencyStatus.execute(
+          { id: agencyId, status: "active" },
+          backofficeJwtPayload,
+        ),
         new NotFoundError(`No agency found with id ${agencyId}`),
       );
     });
 
-    it("returns HTTP 409 if attempt to update to another existing agency (with same address and kind, and a status 'active' or 'from-api-PE'", async () => {
+    it("returns HTTP 409 if attempt to update to another existing agency (with same address and kind, and a status 'active' or 'from-api-PE', (expect if user is admin)", async () => {
```

Comment:
je crois qu'il y a du texte en trop ici:
```suggestion
    it("returns HTTP 409 if attempt to update to another existing agency (with same address and kind, and a status 'active' or 'from-api-PE'", async () => {
```